### ISSUE TITLE
support the 'RK3566' or 'RK3568' soc?

### ISSUE THREAD

**Issue by timiil** [KumaTea/pytorch-aarch64#12]
hello， as title say，the Rockchip had release rk3566 soc (4 * core A55, 0.8 TOPS NPU, Ubuntu 18.04),they also said that it is support caffe，ONNX，pytorch ... framworks.

so，do we have any plan to this SOC on this project? tks.

**Comment by KumaTea** [comment]
Hi.

This project aims to support all ARMv8 64bit processors, so the chips you mentioned should already be supported.

As for the NPU, PyTorch itself does not support GPUs other than Nvidia and AMD.

**Comment by timiil** [comment]
> Hi.
> 
> This project aims to support all ARMv8 64bit processors, so the chips you mentioned should already be supported.
> 
> As for the NPU, PyTorch itself does not support GPUs other than Nvidia and AMD.

thanks for your reply. you mean the soc may be already supported by this project , but without the NPU computing enhancement?

**Comment by timiil** [comment]
> Hi.
> 
> This project aims to support all ARMv8 64bit processors, so the chips you mentioned should already be supported.
> 
> As for the NPU, PyTorch itself does not support GPUs other than Nvidia and AMD.

thanks for your reply. you mean the soc may be already supported by this project , but without the NPU computing enhancement?

**Comment by KumaTea** [comment]
> thanks for your reply. you mean the soc may be already supported by this project , but without the NPU computing enhancement?

Yes. However some boards may contain proprietary NPU chips (`SPR2801S` for example), which _may_ gain significantly performance improvement after **converting PyTorch model to their proprietary formats**.
